src/components/Memoroom.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const MemoRoomContainer = styled.div`
+  display: flex;
+  width: 300px;
+  height: 300px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  border: 1px solid black;
+
+  img {
+    width: 30px;
+    margin: 5px;
+  }
+
+  .image-wrapper {
+    display: flex;
+    visibility: visible;
+    width: 300px;
+    justify-content: flex-end;
```

Comment:
```suggestion
    display: flex;
    justify-content: flex-end;
    visibility: visible;
    width: 300px;
```
해당 부분도 css 컨벤션 맞춰주시면 감사하겠습니다!!

아래 css 부분도 한번 더 확인해주세요~~ ㅎㅎ

css 컨벤션 관련 자료 링크 첨부드립니다.
[https://velog.io/@hikoand/CSS-CSS-Coding-Convention](url)

Reply:
네네!